sis-local/setup.py

```diff
@@ -0,0 +1,40 @@
+import setuptools
```

Comment:
TBH, I hadn't considered actually putting this in this repo, but that seems like a nice easy solution

Reply:
If this gets traction, I would like to put it in its own repo so we can do easier version tagging since that will become a thing. But figured this was a good place to start :)